docs/modules/ROOT/pages/references/architecture/control-api-invitation.adoc

```diff
@@ -0,0 +1,94 @@
+= {controlapi}: Invitation
+
+TIP: This resource implements the xref:references/functional-requirements/portal.adoc#_feature_send_invitations["Send Invitations"] and xref:references/functional-requirements/portal.adoc#_feature_accept_invitations["Accept Invitations"] features.
+
+== Object
+
+A virtual resource is chosen to allow for RBAC filtering for list operations.
+The `Invitation` is stored in a `Secret` resource in the backend.
+It may contain sensitive information such as the invitation token.
+
+.Virtual resource
+[source,yaml]
+----
+apiVersion: users.appuio.io/v1
+kind: Invitation
+metadata:
+  name: E303B166-5D66-4151-8F5F-B84BA84A7559 <1>
+spec:
+  note: "New employee dev1 (Delilah Vernon) starting 2020-04-01" <2>
+  email: "dev1@acme.com" <3>
+  # For billing entity invitations
+  roleBindingRef:
+    apiGroup: "rbac.authorization.k8s.io"
+    kind: "ClusterRoleBinding"
+    name: "be-123-viewer"
+    namespace: "" <4>
+  # OR
+  # For organization invitations
+  roleBindingRef:
+    apiGroup: "rbac.authorization.k8s.io"
+    kind: "RoleBinding"
+    name: "control-api:organization-admin"
+    namespace: "acme" <5>
```

Comment:
The third case is Teams. Shall we add an example too?

Reply:
I'll add an example 👍

Reply:
Added an example and reworked the `Organization` example a bit since it requires adding a member to the `OrganizationMembers` object.